modular_skyrat/modules/customization/modules/clothing/glasses/glasses.dm

```diff
@@ -1,68 +1,19 @@
 /obj/item/clothing/glasses	//Code to let you switch the side your eyepatch is on! Woo! Just an explanation, this is added to the base glasses so it works on eyepatch-huds too
 	var/can_switch_eye = FALSE	//Having this default to false means that its easy to make sure this doesnt apply to any pre-existing items
-	var/current_eye = "_R"	//Added to the end of the icon_state to make this easy code-wise, L and R being the wearer's Left and Right
-	/// Stores the current state of the sprite. This is mostly here for reskin compatibility.
-	var/current_sprite_state
-	/// Stores the current worn state, this like current_sprite_state, this is used for reskin compatibility.
-	var/current_worn_state
-
-/obj/item/clothing/glasses/CtrlClick(mob/user)
-	. = ..()
-	if(.)
-		return
-
-	if(!user.canUseTopic(src, be_close = TRUE, no_dexterity = TRUE, floor_okay = !iscyborg(user)))
-		return
-
-	switcheye()
 
 
 /obj/item/clothing/glasses/examine(mob/user)
```

Comment:
Can probably remove this and the can_switch_eye var from everything that uses it now that the system's gutted

Reply:
I left it there only because upstream didn't have anything to indicate in the examine that you could switch eyes. Otherwise it'd be there.